package-lock.json

```diff
@@ -21026,9 +21329,9 @@
       }
     },
     "terser": {
-      "version": "5.14.2",
-      "resolved": "https://registry.npmjs.org/terser/-/terser-5.14.2.tgz",
-      "integrity": "sha512-oL0rGeM/WFQCUd0y2QrWxYnq7tfSuKBiqTjRPWrRgB46WD/kiwHwF8T23z78H6Q6kGCuuHcPB+KULHRdxvVGQA==",
+      "version": "5.14.1",
```

Comment:
@Ashish-portalFxDev Can you review this part. Seems fishy to me. Not sure why this changes. Rest the infra test cases run fine.

Reply:
Yes we should not downgrade terser version...Please revert this change.

Terser version was purposefully upgraded in this PR: https://github.com/microsoft/powerplatform-vscode/commit/5ccbe7c638e8d6a9713eacba84ef4b5c9b931943